app/static/assets/js/form-validation.js

```diff
@@ -0,0 +1,20 @@
+// Example starter JavaScript for disabling form submissions if there are invalid fields
+(function () {
+  'use strict'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/garethtdavies/mina-taxes/pullRequest?prid=7877435)
```suggestion
  "use strict"
```